lib/docker.js

```diff
@@ -100,7 +108,9 @@ async function resolveCodeUriToMount(codeUri) {
   if (stats.isDirectory()) {
     target = '/code';
   } else {
-    target = path.join('/code', path.basename(codeUri));
+    // could not use path.join('/code', xxx)
+    // in windows, it will be translate to \code\xxx, and will not be recorgnized as a valid path in linux container
+    target = '/code/' + path.basename(codeUri);
```

Comment:
path.posix.join

https://nodejs.org/api/path.html#path_path_posix

Reply:
it will not work on windows? 
path.posix.join source code is here: https://github.com/nodejs/node/blob/master/lib/path.js#L1141-L1158

Reply:
done